mempool/clist_mempool.go

```diff
@@ -389,6 +389,18 @@ func (mem *CListMempool) resCbFirstTime(
 				return
 			}
 
+			// Check transaction not already in the mempool
```

Comment:
It would be helpful to explain why would the transaction be already in the mempool. Maybe, once we have an issue detailing the bug, add the link to the issue here.